lang/cs/Org.Apache.REEF.Common/TeleMetry/Counter.cs

```diff
@@ -0,0 +1,126 @@
+﻿// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+
+namespace Org.Apache.REEF.Common.Telemetry
+{
+    [DataContract]
```

Comment:
We probably should change this later to only serialize the absolute minimum. Right now, it seems like it also serializes `_name` and `_description`, which can get huge. Not a problem for now, but we should sort this out in a future commit.